docs/data/census.md

```diff
@@ -0,0 +1,66 @@
+|Fraction Black |Number of individuals who are black alone divided by total population |2000 Census SF1 100% Data Table P008 |
+|Poverty Rate |Fraction of population below the poverty rate |2000 Census SF3 Sample Data Table P087 |
+|Racial Segregation |Multi-group Theil Index calculated at the census-tract level over four groups: White alone, Black alone, Hispanic, and Other |2000 Census SF1 100% Data Table P008|
+|Income Segregation |Rank-Order index estimated at the census-tract level using equation (13) in Reardon (2011) the δ vector is given in Appendix A4 of Reardon's paper. H(pk) is computed for each of the income brackets given in the 2000 census. See Appendix D for further details. |2000 Census SF3 Sample Data Table P052|
+|Segregation of Poverty (<p25) |H(p25) estimated following Reardon (2011); we compute H(p) for 16 income groups defined by the 2000 census. We estimate H(p25) using a fourth-order polynomial of the weighted linear regression in equation (12) of Reardon (2011). |2000 Census SF3 Sample Data Table P052 |
+|Segregation of Affluence (>p75) |Same definition as segregation of poverty, but using p75 instead of p25 |2000 Census SF3 Sample Data Table P052 |
+|Fraction with Commute < 15 Mins |Number of workers that commute less than 15 minutes to work divided by total number of workers. Sample restricts to workers that are 16 or older and not working at home. |2000 Census SF3 Sample Data Table P031|
+|Logarithm of Population Density |Logarithm of the Population Density where the Population Density is defined as the Population divided by the Land Area in square miles. |2000 Census Gazetteer Files|
+|Household Income per Capita |Aggregate household income in the 2000 census divided by the number of people aged 16-64 |2000 Census SF3 Sample Data Table P054|
+|Labor Force Participation |Share of people at least 16 years old that are in the labor force |2000 Census SF3 Sample Data Table P043|
+|Share Working in Manufacturing |Share of employed persons 16 and older working in manufacturing. |2000 Census SF3 Sample Data Table P049|
+|Fraction Foreign Born |Share of CZ residents born outside the United States |2000 Census SF3 Sample Data Table P021|
+|Fraction of Single Mothers |Number of single female households with children divided by total number of households with children |2000 Census SF3 Sample Data Table P015 |
+|Fraction of Adults Divorced |Fraction of people 15 or older who are divorced |2000 Census SF3 Sample Data Table P018|
+|Fraction of Adults Married |Fraction of people 15 or older who are married and not separated |2000 Census SF3 Sample Data Table P018|
+|Median Monthly Rent |Median "Contract Rent" (monthly) for the universe of renter-occupied housing |2000 Census SF3a (NHGIS SF3a, code: GBG) units paying cash rent |
+|Median House Price |Median value of housing units at the county level (population weighted to CZ level for CZ covariate). |2000 Census SF3a (NHGIS SF3a, code: GB7) |
+
+#### ACS5 2015 API Field Names
+
```

Comment:
It would be useful to add a quick note along the lines of "Census uses code names (like B02001_003E) to uniquely designate specific data fields they provide. We note those field names here, along with a more descriptive title, so that users of Housing Insights can find Census documentation on those fields." Without context it's not necessarily clear what those ids are.